core/src/main/java/bisq/core/payment/validation/AltCoinAddressValidator.java

```diff
@@ -30,14 +34,8 @@
 
 import lombok.extern.slf4j.Slf4j;
 
-
-
-import bisq.asset.AddressValidationResult;
-import bisq.asset.Asset;
-import bisq.asset.AssetRegistry;
-
 @Slf4j
-public final class AltCoinAddressValidator extends InputValidator {
+public class AltCoinAddressValidator extends InputValidator {
```

Comment:
If that was removed for testing consider to use the mockito-extensions like in: https://github.com/bisq-network/bisq/pull/3587/commits/6d983dec8d37a038a6e9f1cc024f941050632140#diff-1bf51dc85c38e8dd66e85a3d5286d1e0